src/overcooked_ai_py/tutorial_notebooks/Agent_evaluator_intoduction.ipynb

```diff
@@ -0,0 +1,1387 @@
+       "    var box_id = \"#graph-div-68167c66-d686-11ea-bb92-f7ccd9451793\" ;\n",
+       "    var settings = {\"height\": 250, \"width\": 720, \"margin\": {\"top\": 20, \"right\": 60, \"bottom\": 180, \"left\": 40}, \"hold_line_width\": 3, \"highlighted_hold_line_width\": 6, \"object_line_width\": 0, \"highlighted_object_line_width\": 3, \"object_event_height\": 10, \"object_event_width\": 16, \"label_text_shift\": 30, \"add_cumulative_data\": true, \"cumulative_data_ticks\": 4, \"show_legends\": true, \"legend_title_size\": 10, \"legend_points_height\": 10, \"legend_points_width\": 16, \"legend_points_margin\": {\"bottom\": 5, \"right\": 5}, \"legend_margin\": {\"right\": 5}} ;\n",
+       "    render_event_chart(data, box_id, settings);\n",
+       "}); </script>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "agent_eval.events_visualization(trajectory_human_pair, traj_index=0, ipython=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Custom layouts\n",
+    "Besides premade layouts found in the[layout directory](https://github.com/HumanCompatibleAI/overcooked_ai/tree/master/overcooked_ai_py/data/layouts) you can create your own layouts to run agents on. Lets first look at example layout:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "corner_circut_grid = {\n",
+    "    \"grid\":  \"\"\"XXXPPXXX\n",
+    "                X  2   X\n",
+    "                D XXXX S\n",
+    "                X  1   X\n",
+    "                XXXOOXXX\"\"\",\n",
+    "    \"start_order_list\": None,\n",
+    "    \"cook_time\": 20,\n",
+    "    \"num_items_for_soup\": 3,\n",
+    "    \"delivery_reward\": 20,\n",
+    "    \"rew_shaping_params\": None\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Layout territory is defined by grid. Every character is one tile. Available tiles are:\n",
+    "- empty space - ' '\n",
+    "- counter - 'X'\n",
+    "- onion dispenser - 'O'\n",
+    "- tomato dispenser - 'T'\n",
+    "- pot (place where players cook soup from onions and tomatoes) - 'P' \n",
+    "- dish dispenser - 'D '\n",
+    "- serving location - 'S'\n",
+    "- player starting location - number  \n",
+    "  \n",
+    "You can save layout in ovecooked_ai/overcooked_ai_py/data/layouts directory and then run agent evaluator AgentEvaluator({\"layout_name\": layout_name}) where layout_name is filename without \".layoyut extension\".  \n",
```

Comment:
This notebook is not up to date with master branch. Probably needs some refinement before merging this file to master - see comment possible changes in  https://github.com/HumanCompatibleAI/overcooked_ai/issues/48 issue